src/scimPatch.ts

```diff
@@ -401,6 +402,21 @@ function assign(obj:any, keyPath:Array<string>, value:any) {
         }
         obj = obj[key];
     }
+
+    // If the attribute is an array and the operation is "add",
+    // then the value should be added to the array
+    const attribute = obj[keyPath[lastKeyIndex]];
+    if (op.toLowerCase() === "add" && Array.isArray(attribute)) {
+        // If the value is also an array, append all values of the array to the attribute
+        if (Array.isArray(value)) {
+            value.forEach((val) => attribute.push(val));
```

Comment:
I would prefer to affect the value to `obj` it make it more readable.

I propose to use something like that:
```suggestion
            obj[keyPath[lastKeyIndex]] = [...attribute, ...value];
```

Reply:
@thomaspoignant 
Right, agree with the readability point, good observation.
Obvious code is always better than smart code which needs to be figured out, which in this case was: since `attribute` is an array, `obj` & `attribute` share the same reference rather than copies. Though correct, this needs a little bit of mental work to figure out.

So yeah, we can go ahead with your suggestion, that's better 👍